### PR TITLE
Fixing bug, adding error handling for uriScoringHelper.

### PR DESCRIPTION
**Pull request by lawzlo**
@stkem @yingjieMiao @leogrim

### CODE REVIEW

File: kifi-backend/modules/common/app/com/keepit/common/healthcheck/Healthcheck.scala [hunk 1]
```diff
@@ -149,7 +149,7 @@ class HealthcheckActor @Inject() (
     case CheckUpdateStatusOfService =>
       val currentDate: DateTime = currentDateTime
       val lastCompilationDate: DateTime = services.compilationTime
-      val betweenDays = Days.daysBetween(currentDate, lastCompilationDate).getDays
+      val betweenDays = Days.daysBetween(lastCompilationDate, currentDate).getDays
```
Comment (stkem) [inline kifi-backend/modules/common/app/com/keepit/common/healthcheck/Healthcheck.scala:152]: Oh, was it a negative number before?

Comment (lawzlo) [inline kifi-backend/modules/common/app/com/keepit/common/healthcheck/Healthcheck.scala:152]: Yes. : (

File: kifi-backend/modules/curator/app/com/keepit/curator/commanders/TopUriSeedIngestionHelper.scala [hunk 2]
```diff
@@ -74,7 +74,7 @@ class TopUriSeedIngestionHelper @Inject() (
       }
     }
 
-    val betweenHours = Hours.hoursBetween(currentDateTime, lastIngestionTime).getHours
+    val betweenHours = Hours.hoursBetween(lastIngestionTime, currentDateTime).getHours
```
Comment (stkem) [inline kifi-backend/modules/curator/app/com/keepit/curator/commanders/TopUriSeedIngestionHelper.scala:77]: Ah, and thats why that is not reingesting? :-)

Comment (lawzlo) [inline kifi-backend/modules/curator/app/com/keepit/curator/commanders/TopUriSeedIngestionHelper.scala:77]: very likely.

**Comment by stkem** [comment]
lgtm
For the future, try to have unrelated changes in separate commits. Makes it easier to find if  there is a problem down the road.

**Comment by lawzlo** [comment]
Cool. I will do that next time if have unrelated changes

**Pull request merged**